src/Commands/AbstractDiffVendorCommand.php

```
@@ -0,0 +1,372 @@
+        }
+
+        // Try array comparison if applicable
+        if ($this->shouldCompareAsArrays()) {
+            try {
+                $vendorArr = include $vendorFile;
+                $targetArr = include $localFile;
+
+                // Verify both includes returned arrays
+                if (is_array($vendorArr) && is_array($targetArr)) {
+                    // Sort recursively to ignore ordering differences
+                    $vendorArr = \Illuminate\Support\Arr::sortRecursive($vendorArr);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Arr' in method 'compareFileContents'. (undefined)

(StaticAccess)